CHANGELOG.yaml

```
@@ -12,6 +12,17 @@ meta:
 
 changes:
 
+  - title: Logger for ES request on /summary endpoint
+    upgrade:
+      - deploy
+
+  - title: Fix file type summary is truncated to ten entries
```

```suggestion
  - title: Fix: file type summary is truncated to ten entries
```

What's the reason for dropping the colon from my [previous](https://github.com/DataBiosphere/azul/pull/536#discussion_r233684382) suggestion?

---

Two test failed with the following error message:
```
yaml.scanner.ScannerError: mapping values are not allowed here
  in "/home/travis/build/DataBiosphere/azul/CHANGELOG.yaml", line 19, column 15
```

---

@hannes-ucsc refer to [travis-logs](https://api.travis-ci.org/v3/job/455622756/log.txt)